presto-base-jdbc/src/main/java/io/prestosql/plugin/jdbc/BaseJdbcClient.java

```diff
@@ -709,15 +708,18 @@ public void rollbackCreateTable(ConnectorSession session, JdbcOutputTableHandle
     }
 
     @Override
-    public String buildInsertSql(JdbcOutputTableHandle handle)
+    public String buildInsertSql(JdbcOutputTableHandle handle, List<WriteFunction> columnWriters)
     {
+        checkArgument(handle.getColumnNames().size() == columnWriters.size(), "handle and columnWriters mismatch: %s, %s", handle, columnWriters);
```

Comment:
```suggestion
        checkArgument(handle.getColumnNames().size() == columnWriters.size(), "columnNames and columnWriters mismatch: %s, %s", handle.getColumnNames(), columnWriters);

```

Reply:
i intentionally log whole handle, hoping to capture more of the useful information